floodlight/models/kinematics.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+
+from floodlight import XY
+from floodlight.core.property import PlayerProperty
+from floodlight.models.base import BaseModel
+
+
+class DistanceModel(BaseModel):
+    """Computations for euclidean distances of all players.
+
+    Upon calling the :func:`~DistanceModel.fit`-method, this model calculates the
+    framewise euclidean distance for each player. The following calculations can
```

Comment:
```suggestion
    framewise Euclidean distance for each player. The following calculations can
```

Reply:
Both comments also apply to the other two classes

Reply:
done